javers-spring/src/main/java/org/javers/spring/aspect/JaversAdvice.java

```diff
@@ -0,0 +1,41 @@
+package org.javers.spring.aspect;
+
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.javers.core.Javers;
+
+/**
+ * @author Pawel Szymczyk
+ */
+public class JaversAdvice implements MethodInterceptor {
+
+    private final Javers javers;
+    private final String author;
+
+    public JaversAdvice(Javers javers, String author) {
+        this.javers = javers;
+        this.author = author;
+    }
+
+    @Override
+    public Object invoke(MethodInvocation invocation) throws Throwable {
+
+        Object retVal = invocation.proceed();
+
+        for (Object arg: invocation.getArguments()) {
+            if (arg instanceof Iterable) {
```

Comment:
co jeśli ktoś nie będzie chciał committować wszystkich arg, np jeśli przekazuje jakieś flagi ...
temat to zastanowienia na później